runtime/server/generate_metrics_view.go

```diff
@@ -461,15 +430,16 @@ func marshalMetricsViewYAML(doc *metricsViewYAML, aiPowered bool) (string, error
 	}
 	buf.WriteString("\n")
 
-	enc := yaml.NewEncoder(buf)
-	enc.SetIndent(2)
```

Comment:
Can we keep two-space indents? It's also what we use for most (all?) YAML files in our own repo